packages/application-components/src/components/dialogs/internals/dialog-container.js

```diff
@@ -43,6 +48,7 @@ const DialogContainer = props => (
         parentSelector={props.getParentSelector}
         ariaHideApp={false}
         style={{
+          // stylelint-disable-next-line selector-type-no-unknown
```

Comment:
Assuming that this is a prop looking like a style prop but actually not being one.